README.md

```diff
@@ -27,19 +27,33 @@ In React, having a component that needs to have state is also a clue that it sho
 Here's an example of a React component whose state changes whenever its child element (a checkbox) is clicked.
 
 ```
```

Comment:
Can you add `js` right after the three backticks (no space) so we get some JS formatting?